notification/docs/IntegrationGuide.md

```diff
@@ -0,0 +1,67 @@
+# Integration of payment into checkout process
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Contents**
+
+  - [Glossary](#glossary)
+  - [Requirements](#requirements)
+      - [for CTP project:](#for-ctp-project)
+      - [for the Node.js:](#for-the-nodejs)
+  - [Environment variables to configure the module:](#environment-variables-to-configure-the-module)
+- [After deployment](#after-deployment)
+  - [Register the endpoint](#register-the-endpoint)
+  - [Note](#note)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Glossary
+In this process, there are 2 parties involved:
+
+**Adyen** - the payment provider which will send notifications
+**Notification module** - hosted service that accepts notifications from Adyen,
+processes and stores on a commercetools platform project.  
+
+## Requirements 
+
+#### for CTP project:
+[An `interfaceInteractions` custom type](../resources/payment-interface-interaction-types.json) should be ensured on the Notification module start
+
+#### for the Node.js:
+The application supports all versions of Node.js starting the LTS version 8.
+
+
+## Environment variables to configure the module:
+Name | Content | Required | Default value
+------------ | ------------- | ------------- | -------------
+CTP_PROJECT_KEY | commercetools project key (you can get in the commercetools Merchant Center) | **YES** |
+CTP_CLIENT_ID | commercetools client ID (you can get in the commercetools Merchant Center) | **YES** |
+CTP_CLIENT_SECRET | commercetools client secret (you can get in the commercetools Merchant Center) | **YES** |
+LOG_LEVEL | bunyan log level (`trace`, `debug`, `info`, `warn`, `error`, `fatal`)| NO | `info`
+PORT | port on which the application will run | NO | 443
+
+Check out the deployment [Best Practices documentation](../../docs/BEST_PRACTICES.md)
+
```

Comment:
Here should be point called `Deployment` which describes which docker container is to be used and how to start it.